docker-compose.yaml

```diff
@@ -18,7 +19,8 @@ services:
       #- ${PWD}/.users.yaml:/config/.users.yaml
     networks:
       - argilla
-
+    volumes:
+      - data:/var/lib/argilla
```

Comment:
The prefix given by docker-compose depends on where docker-compose.yaml is placed. Normally, the parent folder name will be used. If we download this file and place it in folder `local` the volume name will be `local_data`. I prefer adding the service name to the created volume: `argilladata` is fine